Makefile.inc

```diff
@@ -54,6 +55,14 @@ skipclippy:
 
 # Testing. A lot of directories can't test either, yet.
 test:
+	cargo test
 
 skiptest:
 	echo Not doing test for this target
+
+coverage:
+	cargo tarpaulin --out Lcov --output-dir "$(OREBOOT)/coverall"
```

Comment:
This still needs to be addressed. Crates should be self-contained, so remove the `--output-dir` and instead concatenate the resulting files for the main `Makefile` target `citest` after the individual reports were created.
Please also add `coverall/` to `.gitignore`.